[Bug] [deploy on k8s ]  worker server and alert server always crash

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

[Bug] [deploy on k8s ]  worker server and alert server always crash

curl not found in worker 

  Warning  Unhealthy  33m (x641 over 14h)     kubelet  Readiness probe failed: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "curl": executable file not found in $PATH: unknown
  Warning  BackOff    8m12s (x2238 over 14h)  kubelet  Back-off restarting failed container
  Warning  Unhealthy  3m23s (x573 over 14h)   kubelet  Liveness probe failed: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "curl": executable file not found in $PATH: unknown

alert crash without log

 kubelet  Back-off restarting failed container


### What you expected to happen

[Bug] [deploy on k8s ]  worker server and alert server always crash

curl not found in worker 

  Warning  Unhealthy  33m (x641 over 14h)     kubelet  Readiness probe failed: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "curl": executable file not found in $PATH: unknown
  Warning  BackOff    8m12s (x2238 over 14h)  kubelet  Back-off restarting failed container
  Warning  Unhealthy  3m23s (x573 over 14h)   kubelet  Liveness probe failed: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "curl": executable file not found in $PATH: unknown

alert crash without log

 kubelet  Back-off restarting failed container


### How to reproduce

.

### Anything else

.

### Version

dev

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

Can you post more information about alert && worker pods? I have tried deploying on k8s and encountered the same problem.

The error log is below for me.

- Alert server 
Logs in the pod are standard, but Kuboard shows an error 
```
Liveness probe failed:
Readiness probe failed:
Container dolphinscheduler-alert failed liveness probe, will be restarted
then loop

```


- Worker server 
There are three pods that will be created by default, and one pod will not be ready with the error log:
```
[INFO] 2022-05-17 17:51:45.682 +0800 org.apache.curator.framework.state.ConnectionStateManager:[251] - State change: CONNECTED
[WARN] 2022-05-17 17:51:45.777 +0800 org.apache.curator.utils.ZKPaths:[78] - The version of ZooKeeper being used doesn't support Container nodes. CreateMode.PERSISTENT will be used instead.
[INFO] 2022-05-17 17:51:45.926 +0800 ru.yandex.clickhouse.ClickHouseDriver:[42] - Driver registered
[INFO] 2022-05-17 17:51:47.238 +0800 com.zaxxer.hikari.HikariDataSource:[110] - DolphinScheduler - Starting...
[INFO] 2022-05-17 17:51:47.400 +0800 com.zaxxer.hikari.HikariDataSource:[123] - DolphinScheduler - Start completed.
[INFO] 2022-05-17 17:51:52.435 +0800 org.apache.dolphinscheduler.remote.NettyRemotingServer:[146] - NettyRemotingServer bind success at port : 1234
[INFO] 2022-05-17 17:51:52.439 +0800 org.apache.dolphinscheduler.service.task.TaskPluginManager:[93] - Registering task plugin: BLOCKING
[INFO] 2022-05-17 17:51:52.440 +0800 org.apache.dolphinscheduler.service.task.TaskPluginManager:[101] - Registered task plugin: BLOCKING
[INFO] 2022-05-17 17:51:52.443 +0800 org.apache.dolphinscheduler.spi.utils.JSONUtils:[59] - init timezone: sun.util.calendar.ZoneInfo[id="Asia/Shanghai",offset=28800000,dstSavings=0,useDaylight=false,transitions=31,lastRule=null]
[WARN] 2022-05-17 17:51:52.767 +0800 org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext:[591] - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'workerServer': Invocation of init method failed; nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
### The error may exist in class path resource [org/apache/dolphinscheduler/dao/mapper/PluginDefineMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select *         from t_ds_plugin_define         where plugin_name = ? and plugin_type = ?         limit 1
### Cause: org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
; bad SQL grammar []; nested exception is org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
[INFO] 2022-05-17 17:51:52.768 +0800 com.zaxxer.hikari.HikariDataSource:[350] - DolphinScheduler - Shutdown initiated...
[INFO] 2022-05-17 17:51:52.776 +0800 com.zaxxer.hikari.HikariDataSource:[352] - DolphinScheduler - Shutdown completed.
[INFO] 2022-05-17 17:51:52.789 +0800 org.apache.curator.framework.imps.CuratorFrameworkImpl:[955] - backgroundOperationsLoop exiting
[INFO] 2022-05-17 17:51:52.802 +0800 org.apache.zookeeper.ZooKeeper:[693] - Session: 0x100c9993b770000 closed
[INFO] 2022-05-17 17:51:52.804 +0800 org.apache.zookeeper.ClientCnxn:[522] - EventThread shut down for session: 0x100c9993b770000
[INFO] 2022-05-17 17:51:53.107 +0800 org.apache.dolphinscheduler.remote.NettyRemotingClient:[390] - netty client closed
[INFO] 2022-05-17 17:51:53.107 +0800 org.apache.dolphinscheduler.service.alert.AlertClientService:[77] - alter client closed
[INFO] 2022-05-17 17:51:53.139 +0800 org.eclipse.jetty.server.session:[149] - node0 Stopped scavenging
[INFO] 2022-05-17 17:51:53.141 +0800 org.eclipse.jetty.server.handler.ContextHandler:[1153] - Stopped o.s.b.w.e.j.JettyEmbeddedWebAppContext@357f6391{application,/,[file:///tmp/jetty-docbase.1235.4530089619059614642/],STOPPED}
[INFO] 2022-05-17 17:51:53.194 +0800 org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener:[136] - 
Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
[ERROR] 2022-05-17 17:51:53.262 +0800 org.springframework.boot.SpringApplication:[843] - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'workerServer': Invocation of init method failed; nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
### The error may exist in class path resource [org/apache/dolphinscheduler/dao/mapper/PluginDefineMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select *         from t_ds_plugin_define         where plugin_name = ? and plugin_type = ?         limit 1
### Cause: org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
; bad SQL grammar []; nested exception is org.postgresql.util.PSQLException: ERROR: relation "t_ds_plugin_define" does not exist
  Position: 23
        at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:160)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:440)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1796)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:944)
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:918)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
        at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754)
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434)
```

And this pod will restart by Kuboard and will be fine.

--- a/dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java
+++ b/dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java
@@ -63,7 +63,7 @@ public class AlertServer implements Closeable {
      */
     public static void main(String[] args) {
         Thread.currentThread().setName(Constants.THREAD_NAME_ALERT_SERVER);
-        new SpringApplicationBuilder(AlertServer.class).web(WebApplicationType.NONE).run(args);
+        new SpringApplicationBuilder(AlertServer.class).run(args);
     }
 
     @EventListener

